packages/venia-ui/lib/components/Image/image.js

```diff
@@ -91,6 +93,7 @@ const Image = props => {
             type={type}
             width={talonResourceWidth}
             widths={widths}
+            ratio={ratio}
```

Comment:
I see you are sending the `ratio` to the `ResourceImage` component and [receiving in the useResourceImage](https://github.com/magento/pwa-studio/pull/2372/files#diff-ed5db5a909eb7db786654de1573e23b6R25) component, but I don't see the code change to [receive the prop in `ResourceImage`](https://github.com/larsroettig/pwa-studio/blob/2333-image-ratio/packages/venia-ui/lib/components/Image/resourceImage.js#L34) component. Is there something that I am missing here?